invenio_override/templates/invenio_override/contact_us.html

```diff
@@ -18,11 +18,14 @@ <h4>{{ _("Need help?") }}</h4>
   {%- endif %}
 
   <div style="padding-bottom: 10px;">
+    {%- if config.SHOW_RIGHT_CONTACT_EMAIL %}
```

Comment:
```suggestion
    {%- if config.SHOW_RIGHT_CONTACT_EMAIL %}
  <div style="padding-bottom: 10px;">
```
if the contact email is false, we dont need the <div> allocating space in body